scripts/clean.php

```diff
@@ -44,6 +44,9 @@ function getHTTPInfo($url, $followLocation = false)
     $total_to_delete++;
     echo "(" . $total_to_delete . ") deleted: " . $row['longURL'] . PHP_EOL;
   }
+  //Impletement deletion of any short URLS which have not been visited in two years
+  $mysqli->query("DELETE FROM tblURLs WHERE DATE_ADD(lastRedirectDate, INTERVAL 2 YEAR) < CURRENT_TIMESTAMP()");
+  
```

Comment:
Need to adjust `$total_to_delete` to account for rows deleted with old redirects. Hint `$mysqli->affected_rows`

Reply:
Updated. Please see commit 83fe9fd